### PR TITLE
[DOC] Improve appearance of parameter types in API docs

### PR DESCRIPTION
**Pull request by anibalsolon**
## Changes proposed in this pull request

Customization on docs styles to separate the parameter name and type:

From:
image

To:
image

## Documentation that should be reviewed

All functions for workflows might be affected.

### CODE REVIEW

**Comment by effigies** [comment]
I cancelled the tests, since only the [doc build](https://8777-53175327-gh.circle-artifacts.com/0/tmp/src/fmriprep/docs/_build/html/api/index.html) was needed.

This LGTM.

**Comment by oesteban** [comment]
@anibalsolon do you want to be part of the zenodo.json file?

**Comment by anibalsolon** [comment]
Sure! I will add myself there. (lovely thing this '[skip ci]' tag on the commit message)

File: .zenodo.json [hunk 1]
```diff
@@ -116,6 +116,11 @@
       "affiliation": "Neuroscience Program, University of Iowa",
       "orcid": "0000-0002-3339-4857"
     },
+    {
+      "name": "Heinsfeld, Anibal S.",
```
Comment (oesteban) [inline .zenodo.json]: Sorry I wasn't clear. You should insert your name right before Russ. I'll merge as soon as that gets fixed.

Thanks!

Comment (anibalsolon) [reply .zenodo.json]: Yeah, I was in doubt about it, definitely should've asked.

Done!

**Pull request merged**